tembo-operator/src/stacks/templates/document.yaml

```diff
@@ -0,0 +1,84 @@
+name: MongoAdapter
+description: Document-facing workloads on Postgres.
+image: "quay.io/tembo/standard-cnpg:15.3.0-1-1096aeb"
+stack_version: 0.1.0
+appServices:
+  - name: fdb-api
+    image: ghcr.io/ferretdb/ferretdb
+    routing:
+      - port: 27018
+        ingressPath: /ferretdb/v1
+        entryPoints:
+          - ferretdb
+        ingressType: tcp
+    env:
+      - name: FERRETDB_POSTGRESQL_URL
+        valueFromPlatform: ReadWriteConnection
+      - name: FERRETDB_LOG_LEVEL
+        value: debug
+      - name: FERRETDB_STATE_DIR
+        value: '-'
+      - name: FERRETDB_LISTEN_TLS_CERT_FILE
+        value: /certs/tls.crt
+      - name: FERRETDB_LISTEN_TLS_KEY_FILE
+        value: /certs/tls.key
+      - name: FERRETDB_LISTEN_TLS
+        value: :27018
+    storage:
+      volumes:
+        - name: ferretdb-data
+          ephemeral:
+            volumeClaimTemplate:
+              spec:
+                accessModes:
+                  - ReadWriteOnce
+                resources:
+                  requests:
+                    storage: 1Gi
+        - name: ferretdb-certs
```

Comment:
Is this section on secrets necessary? Or only for local testing and development?

Reply:
certs? yes I think its necessary

Reply:
Sorry, my mistake with comment placement, I meant just below:         

 secret:
            secretName: sample-mongo-adapter-server1
        volumeMounts:

Reply:
Can you describe how it was used for local testing and development? I'm not sure what `sample-document.yaml` is...

Reply:
Sure thing -- Running Tembo operator locally leverages a yaml, such as sample-message-queue.yaml. MongoAdapter started off as Document Stack, which lends to sample-document.yaml:
- https://github.com/tembo-io/tembo/blob/main/tembo-operator/yaml/sample-document.yaml
- https://tembo.io/docs/tembo-stacks/local-tembo-operator

Reply:
@sjmiller609 do you remember why we needed to add a secret here in order to get FerretDB to work properly in dev?